borg/helpers.py

```diff
@@ -44,13 +44,27 @@ def have_cython():
     import msgpack
 
 
+# return codes returned by borg command
+# when borg is killed by signal N, rc = 128 + N
+EXIT_SUCCESS = 0  # everything done, no problems
+EXIT_WARNING = 1  # reached normal end of operation, but there were issues
+EXIT_ERROR = 2  # terminated abruptly, did not reach end of operation
```

Comment:
these constants are needed by misc modules and the general policy is to have random small stuff needed from everywhere in helpers as far as i can tell.